streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -612,6 +612,30 @@ private void cleanGlobal(final Properties sslConfig,
         Assert.assertEquals(0, exitCode);
     }
 
+    void assertInputTopicDoesNotExist() throws Exception {
+
+        final Properties streamsConfiguration = prepareTest();
+        final List<String> parameterList = new ArrayList<>(
+                Arrays.asList("--application-id", APP_ID + testNo,
+                        "--bootstrap-servers", bootstrapServers,
+                        "--input-topics", INPUT_TOPIC));
```

Comment:
You need to set a non-existing topic name here.

Reply:
Sure will change it.